vendor/yosensi/yo-360.yaml

```diff
@@ -8,7 +8,7 @@ hardwareVersions:
 # Firmware versions (at least one is mandatory)
 firmwareVersions:
   - # Firmware version
-    version: '3.1.0'
+    version: '3.6.1'
```

Comment:
Please do not change the `firmwareVersions` value once it's already created, this will cause problems.

Reply:
Alright, then I'm just supposed to add new firmware versions if they come out right ?
I'll do it on each device and make a second push.

Reply:
Yes please add newer firmware versions instead of replacing them. You can add them to this PR if you wish.

Reply:
Alright will do, although I've got one issue then because I took this task from a previous co-worker and when he made the previous PR the hardware version of yo-people-counter he wrote was 3.0 while at this moment in time we are at 1.1. Is there any way to fix his mistake?

Reply:
I am not sure unfortunately, my co-worker @johanstokking might know.

Reply:
I made a new commit, for now I put back the bad hardware version as numeric 1, and the new one as numeric 2. If it's not possible to change, it can stay like this I suppose.

Reply:
Yes apologies, existing devices in the stack will cease to function if the existing value is changed since the firmware version wouldn't be the same anymore, we ran into this problem a while ago.

Reply:
Alright so let's leave it as is? I've added the new 'proper' version as the second one (in the Yo-People Counter) and it should be good to go

Reply:
Yes that is good. My colleague @Jaime-Trinidad will look at it further and decide if its good, then he'll merge it.